pocs/utils/google/clouddb.py

```diff
@@ -3,42 +3,43 @@
 from warnings import warn
 
 import psycopg2
```

Comment:
Please change the name of the path & file to something other than pocs/utils/google/clouddb.py. I get that it is being used to access a Google offering of postgres, but it is being accessed using a generic postgres library, so how about just calling it pocs/utils/db/postgres.py?

Reply:
Funny how it evolved to that point and I didn't even realize it. Will change, thanks.